### PR TITLE
Fix/usd values

### PR DESCRIPTION
**Pull request by hochbruj**
https://trello.com/c/f5q4JmfI

### CODE REVIEW

File: components/CapBar/index.js [hunk 1]
```diff
@@ -28,15 +28,16 @@ const CapBar = ({
...
           {copies.current}
         </SecondaryText>
         <Title fontSize={statsConfig.fontSize} lineHeight={20}>
           {loading
-            ? 'Loading...'
-            : `${displayCurrent ? displayCurrent : `${formatAmount(current)} VEX-VET`
+            ? "Loading..."
+            : `${
+                displayCurrent ? displayCurrent : `$${formatAmount(current)}`
               }`}
```
Comment (OliverNChalk) [inline components/CapBar/index.js]: While I prefer `"` to `'`, this multiline split is dubious. I assume this was your local prettier running again?

**Review by OliverNChalk — COMMENTED** [review]
Do we want all these lint changes mixed in? Makes it hard to review the PR

**Comment by andersonaguiar** [comment]
image

am I millionaire?

**Review by andersonaguiar — APPROVED** [review]
Nice work man!

**Review by KennethAshley — APPROVED** [review]
Good to merge after we fix the linting

**Pull request merged**